src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -268,10 +269,27 @@ export class FacetValueSuggestions extends Component {
   }
 
   private mapFacetValueSuggestion(resultToShow: IFacetValueSuggestionRow, omnibox: Omnibox) {
-    return <IOmniboxSuggestion>{
+    const field: ISuggestionField = {
+      name: this.options.field.toString(),
+      value: resultToShow.value
+    };
+
+    const suggestion = <IOmniboxSuggestion>{
       html: this.buildDisplayNameForRow(resultToShow, omnibox),
-      onSelect: () => this.onRowSelection(resultToShow, omnibox)
+      text: resultToShow.keyword.text,
+      field: !this.options.isCategoryField
+        ? field
+        : <ISuggestionCategoryField>{
```

Comment:
I'm not a fan of this cast. Could we simplify the `field` property to a string array? If a categoryField is configured, then we split on the delimiting character similar to the logic on lines 310-312?

Reply:
Good point, I'll remove `ISuggestionCategoryField` and use a delimiting character instead.